jss.spec

```diff
@@ -124,10 +124,7 @@ export CFLAGS
 modutil -dbdir /etc/pki/nssdb -chkfips true | grep -q enabled && export FIPS_ENABLED=1
 
 # The Makefile is not thread-safe
-%cmake \
-    -DJAVA_HOME=%{java_home} \
```

Comment:
At least `-DJAVA_HOME` is used by [`cmake`'s `FindJAVA`](https://cmake.org/cmake/help/latest/module/FindJava.html) library.

Reply:
I think `FindJNI` might use the other, but I'm not entirely sure.

Reply:
@cipherboy Thanks, I updated the PR to do something completely different now :)

The ultimate goal is to modify the `build.sh` to call CMake directly instead of `rpmbuild`, then modify the `jss.spec` to call `build.sh` instead of CMake. That way the upstream JSS can be built with `build.sh` without depending on Fedora-specific packaging.

Reply:
Ah nifty :) That's a good idea!